assets/js/choice.js

```diff
@@ -90,16 +90,15 @@ var CHoice;
 
         handleGetCourseExamResponse: function (data) {
             var rows = "";
+            const months = [Januari,Februari,March,April,May,June,Juli,August,September,October,November,December]
 
             if (data.length > 0) {
                 $.each(data, function (i, exam) {
                     var row = CHoice.settings.blueprintRow.split("{% courseCode %}").join(exam.course.code);
 
-                    var month = exam.date.month.toLowerCase().replace(/\b[a-z]/g, function (letter) {
-                        return letter.toUpperCase();
-                    });
+                    var month = months[(parseInt(exam.date.split('-')[1])-1) %11]
```

Comment:
is die `%11` echt nodig, een maand kan toch niet >12 zijn?

Reply:
Nee niet echt. Ik deed het ook meer als een soort failsafe, maar ik zal het weghalen

Reply:
Removed!